Tests/Brokerages/Kraken/KrakenBrokerageTests.cs

```diff
@@ -0,0 +1,426 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using Moq;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using QuantConnect.Brokerages;
+using QuantConnect.Brokerages.Kraken;
+using QuantConnect.Interfaces;
+using QuantConnect.Orders;
+using QuantConnect.Configuration;
+using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading;
+
+// commented because it causes compile errors. This test was copied from GDAX unit tests and had 0 work done on. It's here for a future reminder of "TODO"
+/*namespace QuantConnect.Tests.Brokerages.Kraken
+{
+    
+    [TestFixture]
+    public class KrakenBrokerageTests
+    {
+        //! TODO - treba je vse example API response dol iz krakena potegnit
```

Comment:
Please keep comments in english :)

I think the spirit of this test is good :)